tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -368,6 +370,7 @@ def test_get_compute_worker_state_and_message_OPEN() -> None:
         state=DockerRunScheduledState.OPEN,
         created_at=0,
         last_modified_at=1,
+        runs_on=[6]
```

Comment:
I am going to put on my Malte hat and ask you to write tests :)

Reply:
Good point, I added them in the commit `added tests for schedule_compute_worker_run`